inorbit_republisher/CHANGELOG.rst

```diff
@@ -0,0 +1,30 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I think we should skip this file entirely. I don't think it should get copied from one version of ROS to the other, it should start fresh. In any case, it doesn't matter for what we need it now. Let's not copy it for now.